lib/airbrake-ruby.rb

```diff
@@ -240,5 +241,23 @@ def close
     def create_deploy(deploy_params)
       @notifiers[:default].create_deploy(deploy_params)
     end
+
+    ##
+    # Returns current thread context. Can be used to set new thread context
+    # values. The thread context is cleared on every `notify/notify_sync` call,
+    # and also and when the current thread `join`'s. The values assigned to the
+    # thread context are passed to the `params` field of a notice.
+    #
+    # @example
+    #   Airbrake.thread_context[:bingo] = :bango
+    #
+    #   # This call will deliver `{ :bingo => :bango }` to the dashboard.
+    #   Airbrake.notify('Oops')
+    #
+    # @since v2.1.0
+    # @return [Airbrake::ThreadContext] the thread context of
+    def thread_context
```

Comment:
I am worried that `.context` will clash with our naming for `context` payload (`notice[:context]`), hence that's why I chose `thread_context`. I can see how it's leaking, though. Any other ideas? Do you happen to know if anyone else besides Honeybadger have this feature (and how it's called there)?